serialization_spec/serialization.py

```diff
@@ -261,8 +280,9 @@ def get_object(self):
 
     def get_queryset(self):
         queryset = self.queryset
-        if hasattr(self, 'get_serialization_spec'):
-            self.serialization_spec = self.get_serialization_spec()
+        self.serialization_spec = get_serialization_spec(self)
+        if self.serialization_spec is None:
+            raise Exception('SerializationSpecMixin requires serialization_spec or get_serialization_spec')
```

Comment:
Might be nice to raise a more descript exception here, eg `django.core.exceptions.ImproperlyConfigured`?